components/automate-gateway/api/iam/v2/policy.proto

```diff
@@ -14,6 +15,38 @@ import "components/automate-grpc/protoc-gen-policy/api/annotations.proto";
 import "components/automate-grpc/protoc-gen-policy/iam/annotations.proto";
 
 service Policies {
+  /*
+  Create a new policy
+
+  Creates a new IAM policy used to control permissions in Automate.
+  A policy is composed of one or more statements that act on a set of members.
+  Each statement contains a role as well as a list of projects.
+
+  The role defines a set of actions that the statement is scoped to.
+  The project list defines the set of resources that the statement is scoped to.
+  Pass `"projects": ["*"]` to scope a statement to every project.
+
+  The top-level projects list for the policy defines which project(s) the policy belongs to,
+  whereas the statement level projects list defines which project(s) the statement applies to.
+
+  This example creates a new policy in the (unassigned) project that grants the viewer role
+  on a few projects for all LDAP teams:
+
+  ```
+  {
+    "name": "My Viewer Policy",
```

Comment:
maybe we could use a different example, maybe add another statement with some in-line actions? Since this kind of Project-Viewer policy would be generated when the project is created (though not for both projects I spose)

Reply:
Been trying to avoid inline actions but I'll add another statement with a custom role.